web/router.ex

```diff
@@ -75,6 +75,7 @@ defmodule CoursePlanner.Router do
     end
 
     resources "/settings", SettingController, only: [:show, :edit, :update], singleton: true
+    resources "/aboutclientprogram", AboutClientProgramController, only: [:show], singleton: true
```

Comment:
oki i change it to about and if there will be any other need we can decide later